secrets.yaml.example

```diff
@@ -0,0 +1,2 @@
+Google:
+    API_KEY: 
```

Comment:
Theoretically we should not need a Google API key on the frontend. What's this being used for?

Reply:
I'm using it for my docker-compose file which spins up the backend for me to test with, the file is provided for the container. I left the compose file in there so anyone else who builds on this project would have an easier time setting up the environment